option.go

```diff
@@ -35,16 +35,19 @@ var (
 		WithPolicyErrRetryInterval("1m"),
 		WithPubkeyErrRetryInterval("1m"),
 		WithJwkErrRetryInterval("1m"),
-		WithATProcessorParams(NewATProcessorParam(true, "1h", "1h")),
+		WithATProcessorParams(NewATProcessorParam(true, true, false, nil, "1h", "1h")),
 		WithRTVerifyRoleToken(true),
 		WithRCVerifyRoleCert(true),
 	}
 )
 
 type ATProcessorParam struct {
-	verifyCertThumbprint bool
-	certBackdateDur      string
-	certOffsetDur        string
+	verifyOAuth2AccessToken bool
+	verifyCertThumbprint    bool
+	verifyTokenClientID     bool
+	authorizedPrincipals    map[string][]string
```

Comment:
`authorizedPrincipals `/`authorizedClientIDs` which seems better? 😵

Reply:
Fixed to correspond to verifyTokenClientID.

d6e400e